.github/workflows/build-go.yml

```diff
@@ -4,8 +4,20 @@ on:
   merge_group:
   push:
     branches: [main]
+    paths:
+      - 'operator/**'
+      - 'aggregator/**'
+      - 'common/**'
+      - 'core/**'
+      - '.github/workflows/build-go.yml'
   pull_request:
     branches: ["*"]
+    paths:
```

Comment:
Missing
- metrics

Reply:
Solved in [20793fe](https://github.com/yetanotherco/aligned_layer/pull/763/commits/20793fe4eef1b7d6eb1b59ad662cfae1a6938925)